aemedge/blocks/author-profiles/author-profiles.js

```diff
@@ -8,14 +8,15 @@ async function addAuthorProfiles(block, keys) {
   if (entries && entries.length) {
     if (keys.length > 1) {
       block.classList.add(`elems${keys.length}`);
-      entries.forEach((entry) => {
-        const avatar = new Avatar(entry.title, entry.description, entry.path, entry.image);
+      entries.forEach((e) => {
+        const avatar = new Avatar(e.author, e.title, e.description, e.path, e.image);
```

Comment:
I think you could use destructuring for this - might look a little nicer.

Reply:
I have refactored the avatar-from-authorentry code differently. Please review.